runtime/doc/lua.txt

```diff
@@ -4146,6 +4147,13 @@ Provides operations to compare, calculate, and convert ranges represented by
                      See |Range:lsp()|.
 
 
+Range:empty()                                                  *Range:empty()*
```

Comment:
I _think_ (at least from [this other reference](https://github.com/ribru17/neovim/blob/8b2d27aa6acef1b298bf6350ff13cfba8f9c62f3/runtime/doc/lua.txt#L4509)) (and it's also my personal preference) to prefix methods/functions that return a boolean with `is_`.

Reply:
We are lacking guidance on this, but we have existing `empty()` and `isempty()` so another variant is probably not best. And we tend to prefer noodlecase for Lua stdlib.